participant/e2e/routes/confirm.spec.ts

```diff
@@ -85,5 +85,29 @@ test("hitting the back button after submission shows a warning", async ({
   const startOverLink = page.getByRole("link", { name: "Start over" });
   await expect(startOverLink).toHaveCount(1);
   await startOverLink.click();
-  await expect(page).toHaveURL("/gallatin/recertify/?newSession=true");
+  await expect(page).toHaveURL("/gallatin/recertify");
```

Comment:
Shouldn't this include the query param `newSession=true`?

Reply:
No, because the cookies code redirects you to the root after obeying the `newSession` directive and resetting your session

Reply:
Oh I see! Thank you.

Reply:
```typescript
  const resetSession: boolean = url.searchParams.get("newSession") === "true";
  let forceRedirect: boolean = resetSession;
```

Forceredirect is always true if resetsetssion is true